src/devices/lonsonho.ts

```diff
@@ -217,6 +217,53 @@ const definitions: Definition[] = [
             e.binary('calibration', ea.ALL, 'ON', 'OFF'), e.binary('motor_reversal', ea.ALL, 'ON', 'OFF'),
             e.numeric('calibration_time', ea.STATE).withUnit('S').withDescription('Calibration time')],
     },
+    {
+        fingerprint: [{modelID: 'TS0603', manufacturerName: '_TZE600_wxq8dpha\u0000'}],
+        model: 'VM-Zigbee-S02-0-10V',
+        vendor: 'Lonsonho',
+        description: '2CH Zigbee 0-10V Dimmer Module',
+        fromZigbee: [tuya.fz.datapoints],
+        toZigbee: [tuya.tz.datapoints],
+        exposes: [
+            tuya.exposes.lightBrightnessWithMinMax().withEndpoint('l1'),
+            tuya.exposes.lightBrightnessWithMinMax().withEndpoint('l2'),
+            tuya.exposes.countdown().withEndpoint('l1'),
+            tuya.exposes.countdown().withEndpoint('l2'),
+            tuya.exposes.switchType().withEndpoint('l1'),
+            tuya.exposes.switchType().withEndpoint('l2'),
+            e.power_on_behavior(['off', 'on', 'previous']).withAccess(ea.STATE_SET),
+        ],
+        endpoint: (device) => {
+            return {'l1': 1, 'l2': 1, 'l3': 1};
+        },
+        meta: {
+            multiEndpoint: true,
+            // All datapoints go in here
+            tuyaDatapoints: [
+                [1, 'state_l1', tuya.valueConverter.onOff],
+                [2, 'brightness_l1', tuya.valueConverter.scale0_254to0_1000],
+                [3, 'min_brightness_l1', tuya.valueConverter.scale0_254to0_1000],
+                [4, 'switch_type_l1', tuya.valueConverterBasic.lookup({
+                    'momentary': new tuya.Enum(0),
+                    'toggle': new tuya.Enum(1),
+                    'state': new tuya.Enum(2),
+                })],
+                [5, 'max_brightness_l1', tuya.valueConverter.scale0_254to0_1000],
+                [6, 'countdown_l1', tuya.valueConverter.raw],
+                [7, 'state_l2', tuya.valueConverter.onOff],
+                [8, 'brightness_l2', tuya.valueConverter.scale0_254to0_1000],
+                [9, 'min_brightness_l2', tuya.valueConverter.scale0_254to0_1000],
+                [10, 'switch_type_l2', tuya.valueConverterBasic.lookup({
+                    'momentary': new tuya.Enum(0),
```

Comment:
Can you create a new value converter for this here: https://github.com/Koenkk/zigbee-herdsman-converters/blob/c78fad16e46648156b25dcec877b00642b9f1dca/src/lib/tuya.ts#L441